app/services/documents.js

```diff
@@ -0,0 +1,39 @@
+const { format, parseISO } = require('date-fns');
+const { getDocuments } = require('../api/documents')
+
+class DocumentsService {
```

Comment:
We don't tend to use classes unless we can help it. E.g. in the rare case we need EventEmitter.

Could you export the functions for a regular module?

Reply:
refactored.